azure-kusto-data/tests/async_unittest.py

```diff
@@ -0,0 +1,11 @@
+import unittest
```

Comment:
please use pytest (unitest code is legacy code)

Reply:
I agree, pytest is way better... But I wanted to keep the same writing style